automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala

```diff
@@ -57,7 +58,25 @@ class WorkspacesAzureApiSpec extends AnyFlatSpec with Matchers with BeforeAndAft
     }
   }
 
-  "Rawls" should "allow creation and deletion of azure workspaces" in {
+  "Other Terra services" should "include Leonardo" in {
```

Comment:
Moved these API tests up to the top so it's easier to tell if we don't have access to a service.